DQMOffline/Trigger/python/WprimeMonitor_cff.py

```diff
@@ -0,0 +1,93 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.WprimeMonitor_cfi import hltWprimemonitoring
+
+WprimeEle115 = hltWprimemonitoring.clone()
+WprimeEle115.FolderName = cms.string('HLT/Wprime/WprimeEle115')
+WprimeEle115.nmuons = cms.uint32(0)
+WprimeEle115.nelectrons = cms.uint32(1)
+WprimeEle115.njets = cms.uint32(0)
+WprimeEle115.eleSelection = cms.string('pt>50 & abs(eta)<2.1 & (dr03TkSumPt+dr04EcalRecHitSumEt+dr04HcalTowerSumEt)/pt<0.1')
```

Comment:
pt cut is only 50 GeV which is good enough as confirmed by analyzers, we are looking at such a high pt electron path. And regarding eta paths they want it to change to 2.4. I will change that .